### ISSUE TITLE
Babel plugin fails

### ISSUE THREAD

**Issue by asbjornh** [Creuna-Oslo/prop-types-csharp-webpack-plugin#21]
Not sure why.

```
ERROR in ./source/components/page-layout/page-layout.jsx 45:50
Module parse failed: Unexpected token (45:50)
You may need an appropriate loader to handle this file type.
| };
|
> PageLayout.Content = ({ className, children }) => <div className={(0, _classnames2.default)('page-layout--content', className)}>{children}</div>;
|
| PageLayout.Content.propTypes = {
 @ ./source/components/page-layout/index.js 7:18-42
 @ ./node_modules/babel-loader/lib!./node_modules/eslint-loader!./source/app.components.js
 @ ./node_modules/expose-loader?Components!./source/app.components.js-exposed
 @ multi babel-polyfill formdata-polyfill whatwg-fetch ./source/js/polyfills/nodelist-foreach.js ./source/js/input-detection-loader expose-loader?React!react expose-loader?ReactDOM!react-dom expose-loader?Components!./source/app.components.js
```

.babelrc
```
{
  "plugins": [
    "babel-plugin-lodash",
    "babel-plugin-transform-class-properties",
    "babel-plugin-transform-object-rest-spread",
    "@creuna/prop-types-csharp-webpack-plugin/babel-plugin"
  ],
  "presets": [
    [
      "env",
      {
        "debug": false,
        "targets": {
          "browsers": ["last 2 versions"]
        },
        "useBuiltIns": true
      }
    ],
    "react"
  ]
}
```

Webpack jsx rule
```
        {
          test: /\.jsx?$/,
          exclude: /node_modules/,
          use: ['babel-loader', 'eslint-loader']
        },
```

**Comment by asbjornh** [comment]
Also, babel versions:

```
    "babel-core": "^6.26.0",
    "babel-eslint": "^8.1.2",
    "babel-loader": "^7.0.0",
    "babel-plugin-lodash": "^3.3.2",
    "babel-plugin-syntax-jsx": "^6.18.0",
    "babel-plugin-syntax-object-rest-spread": "^6.13.0",
    "babel-plugin-transform-class-properties": "^6.24.1",
    "babel-plugin-transform-es2015-shorthand-properties": "^6.24.1",
    "babel-plugin-transform-object-rest-spread": "^6.26.0",
    "babel-polyfill": "^6.26.0",
    "babel-preset-env": "^1.6.1",
    "babel-preset-react": "^6.23.0",
    "babel-register": "^6.26.0",
```

**Comment by asbjornh** [comment]
Turns out there is some strange thing that happens when you call `path.skip()` in an `AssignmentExpression` visitor visiting a Stateless React component that has another component defined like this:

```
const Component = () => <div />;
Component.ChildComponent = () => <div />;
```

Removing call to `path.skip()` fixes the issue

**Comment by asbjornh** [comment]
Fixed in `0.1.5`
